sdk/notificationhubs/notification-hubs/package.json

```diff
@@ -24,18 +24,19 @@
     "clean": "dev-tool run vendored rimraf --glob dist dist-* temp types *.tgz *.log",
     "extract-api": "dev-tool run build-package && dev-tool run extract-api",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
-    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "integration-test:browser": "npm run build:test && dev-tool run test:vitest --browser -- -c vitest.browser.int.config.ts",
-    "integration-test:node": "dev-tool run test:vitest --esm",
```

Comment:
What about the live-test or unit-test pipelines that depend on these integration-test/unit-test commands (during this transition)?

Maybe just reference it as `npm run test:node` until we can completely remove and update the centralized CI?